src/PeoplePage.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable prettier/prettier */
+import { useEffect, useState } from 'react';
+import { Loader } from './components/Loader';
+import { Person } from './types';
+import { TableComponent } from './components/TableComponent';
+import { getPeople } from './api';
+
+export function PeoplePage() {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+
+  function fetchPeople() {
+    getPeople().then(person => {
+      person ? setPeople(person) : setIsLoadingError(true);
+    })
+      .catch(() => setIsLoadingError(true))
+      .finally(() => setIsLoading(false));
+  }
+
+  useEffect(() => {
+    fetchPeople();
+    setIsLoading(true);
```

Comment:
Consider setting `setIsLoading(true)` before calling `fetchPeople()` to ensure that the loading state is set immediately when the component mounts. This will prevent any delay in showing the loading indicator.